pom.xml

```diff
@@ -14,7 +14,7 @@
   <packaging>pom</packaging>
 
   <properties>
-    <nuxeo.target.version>11.*</nuxeo.target.version>
+    <nuxeo.target.version>[11.3,11.6)</nuxeo.target.version>
```

Comment:
why specifying an upper limit?

Reply:
I'm not sure what strategy should be chosen here actually: I'm planning on making some changes to the runtime, that i could use on explorer side, during the 11.5.x lifecycle, and at this point, the package will not be compatible with 11.5.x anymore so I'd rather not have the current version match 11.6 for instance.

That being said, most recent explorer releases match "server-11.*" whereas they're only compatible with >= 11.3, and may not be compatible with 11.6.

Note i would have loved if using `[11.3,11.5)` made it compatible with 11.5.2 for instance, but that's not the case.